fdbrpc/ReplicationUtils.cpp

```diff
@@ -158,6 +170,11 @@ bool findBestPolicySet(
 	return bSucceeded;
 }
 
+/**
+ * findBestUniquePolicySet takes mainluy localityUniquenessKey. Random unique items
+ * are compared with results, the output is returned.
+ **/
```

Comment:
What's a `localityUniquenessKey`? :S   What does `true` or `false` mean as a comparison?  Is the output "returned" though `bestResults`?  How have we survived this long with no comments on these functions?

I didn't realize this is almost the same as `findBestPolicySet` until staring at the code in detail.  You could probably just say that this is "The same as findBestPolicySet, except selected localitySet entries must be unique on localityUniquenessKey".  Which I think is true, but honestly I'm unsure.

Or s/mainluy/mainly/ ?